drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -456,9 +456,22 @@ function DragAndDropEditBlock(runtime, element, params) {
                             var $el = $(el),
                                 name = $el.find('.item-text').val(),
                                 imageURL = $el.find('.item-image-url').val(),
+                                expandedImageURL,
                                 imageDescription = $el.find('.item-image-description').val();
 
                             if (name.length > 0 || imageURL.length > 0) {
+                                if (imageURL) {
+                                    var handlerUrl = runtime.handlerUrl(element, 'expand_static_url');
+                                    $.ajax({
+                                        type: 'POST',
+                                        url: handlerUrl,
+                                        data: JSON.stringify(imageURL),
+                                        success: function (result) {
+                                            expandedImageURL = result.url;
+                                        },
+                                        async: false
```

Comment:
@bradenmacdonald You are completely right, we don't need this here.  I didn't read the code correctly, and somehow assumed we need this because the XBlock gets updated when it is edited (but in fact it is completely reloaded), and in case of the target image we only need this for the preview, so I removed it.  (I didn't like the synchronous call either, but I'm glad I didn't spend much time figuring out a way to avoid it.)